### DISCUSSION TITLE
[Feature] the same functionality when searching a phrase in gallery

### DISCUSSION THREAD

**Discussion by krzemyczek** [immich-app/immich#12343] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If a search in my gallery for "beach" there is no timeline, division of photos into days and no option to multiple select using shift key. 

**Normal Gallery:**
![Screenshot from 2024-09-05 09-33-01-obfuscated](https://github.com/user-attachments/assets/f3107721-1581-4679-b1b3-f83cc74b4e3d)

**Gallery after searching:**
![Screenshot from 2024-09-05 09-33-17-obfuscated](https://github.com/user-attachments/assets/89c2af11-3cd6-4f3b-bdd6-563f25b2f6fc)

solution for this would be to just show photos as in normal gallery view

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8523/#9159

**Reply by krzemyczek** [reply]
thanks! it's not easy find what you look for
